_posts/en/2019-10-01-btse-exchange-operation.md

```diff
@@ -0,0 +1,30 @@
+
+Although exchanges using BIP32 can keep their private key offline, that private key is still a single point of failure. Happily, they can lessen the impact of compromise of any one single key by constructing a k-of-n multisig address with each of the n public keys derived from a different extended public key. What BTSE achieves with this combination is on-the-fly generation of arbitrarily-many addresses that all deposit straight into a multisig cold wallet without needing to touch the private keys.
+
+Aside from BIP32 and multisig, BTSE also uses [P2SH-P2WSH](https://bitcoincore.org/en/segwit_wallet_dev/#complex-script-support) (P2SH-wrapped segwit) for their deposit addresses. Segwit allows BTSE to reduce the fee of spending transactions by reducing their weight, such as for moving to hot wallets or UTXO consolidation. As an example, spending from a P2SH-P2WSH 2-of-3 multisig address [saves 44% when compared to spending from a legacy P2SH one](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#P2SH-wrapped_segwit). As adoption of sending to native segwit P2WSH addresses rises (tracked on Optech’s [compatibility matrix](https://bitcoinops.org/en/compatibility/)), businesses should also consider using these addresses for deposits to [unlock 14% additional savings](https://en.bitcoin.it/wiki/Techniques_to_reduce_transaction_fees#Native_segwit) and [increase security](https://bitcoincore.org/en/2016/01/26/segwit-benefits/#increased-security-for-multisig-via-pay-to-script-hash-p2sh) (256-bits rather than 160-bits) for their multisig scripts. You can learn about [adoption speed](https://bitcoinops.org/en/bech32-sending-support/#adoption-speed), [address security](https://bitcoinops.org/en/bech32-sending-support/#address-security), and much more in our [Bech32 Sending Support series](https://bitcoinops.org/en/bech32-sending-support/).
+
+BIP32, multisig, and segwit work together to ease the burden of operating a secure, usable, and low-fee exchange for BTSE. Optech encourages new exchanges to consider these technologies when designing their infrastructure. For existing exchanges, as noted in a previous field report, a good time to consider adopting these technologies and reducing your technical debt is when you're also consolidating UTXOs.
```

Comment:
Id order as 

segwit, BIP32, and multisig

since that is how we originally started the article. Keep them consistent/tied together.